homeassistant/components/smlight/update.py

```diff
@@ -166,6 +166,10 @@ def _update_done(self) -> None:
             remove_cb()
         self._unload.clear()
 
+        self._attr_in_progress = False
+        self._attr_update_percentage = None
+        self.async_write_ha_state()
```

Comment:
Reset of `_attr_in_progress` was missing, I think that's a bug?

Reply:
As pointed out by @tl-sl, it's already reset in the base entity:
https://github.com/home-assistant/core/blob/5b2113c43da1d17dc09b47d23d5542d3860884cb/homeassistant/components/update/__init__.py#L505